app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt

```diff
@@ -67,11 +67,13 @@ class LoginFragment : Fragment() {
         })
 
         loginViewModel.error.observe(this, Observer {
-            Toast.makeText(context, it, Toast.LENGTH_LONG).show()
+            Snackbar.make(getActivity()!!.findViewById(android.R.id.content),
+                it, Snackbar.LENGTH_LONG).show()
         })
 
         loginViewModel.loggedIn.observe(this, Observer {
-            Toast.makeText(context, getString(R.string.welcome_back), Toast.LENGTH_LONG).show()
+            Snackbar.make(getActivity()!!.findViewById(android.R.id.content),
```

Comment:
use safe calls instead of `!!`

Reply:
but then it is showing type mismatch as snackbar requires view of type View and we are giving View? Pls guide

Reply:
First of all, don't use findViewById. Secondly, that'll fix it

Reply:
`android.R.id.content` This isn't to be used. Use coordinator layout of the current fragment/activity

Reply:
Yeah, OK. Use CoordinatorLayout of the root element

Reply:
but how to get CoordinatorLayout of the root element. when I try to use CoordinatorLayout of root element i.e rootLayout(activity_main.xml) It is becoming null
I'm facing
`java.lang.IllegalStateException: rootView.findViewById(R.id.rootLayout) must not be null
        at org.fossasia.openevent.general.order.OrdersUnderUserFragment.onCreateView(OrdersUnderUserFragment.kt:48)`
Need help

Reply:
Call findViewById on activity